app/tests/integration/nimbus/pages/audience.py

```diff
@@ -0,0 +1,68 @@
+from nimbus.pages.base import Base
+
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.select import Select
+from selenium.webdriver.support import expected_conditions as EC
+
+
+
+class AudiencePage(Base):
+    """Experiment Audience Page."""
+    
+    _page_wait_locator = (By.CSS_SELECTOR, "#PageEditAudience")
+    _channel_select_locator = (By.CSS_SELECTOR, "#channel")
+    _min_version_select_locator = (By.CSS_SELECTOR, "#minVersion")
+    _targeting_select_locator = (By.CSS_SELECTOR, "#targeting")
+    _population_fill_locator = (By.CSS_SELECTOR, "#populationPercent")
+    _expected_clients_locator = (By.CSS_SELECTOR, "#totalEnrolledClients")
+    _enrollment_period_locator = (By.CSS_SELECTOR, "#proposedEnrollment")
+    _duration_locator = (By.CSS_SELECTOR, "#proposedDuration")
+    _page_wait_locator = (By.CSS_SELECTOR, "#PageEditAudience")
+    _save_continue_btn_locator = (By.CSS_SELECTOR, "#save-and-continue-btn")
+
+    def wait_for_page_to_load(self):
+        self.wait.until(EC.presence_of_element_located(self._page_wait_locator))
+
+        return self
+
+    def save_and_continue(self):
+        element = self.selenium.find_element(*self._save_continue_btn_locator)
+        element.click()
+        from nimbus.pages.review import ReviewPage
+
+        return ReviewPage(self.driver, self.base_url).wait_for_page_to_load()
+
+    def set_channel(self, channel="Nightly"):
+        el = self.find_element(*self._channel_select_locator)
+        select = Select(el)
+        select.select_by_visible_text(channel)
+
+    def set_min_version(self,  version=80):
+        el = self.find_element(*self._min_version_select_locator)
+        select = Select(el)
+        select.select_by_value(f"FIREFOX_{version}")
+
+    def set_targeting(self, targeting="US_ONLY"):
+        el = self.find_element(*self._targeting_select_locator)
+        select = Select(el)
+        select.select_by_value(targeting)
```

Comment:
Maybe we could use the same setter/getter pattern everywhere?  So like the getter/setter same name with @thing.setter like below, just for consistency?

Reply:
Hmm yeah, wonder what I was thinking here lol